website/docs/d/iam_trusted_profiles.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_iam_trusted_profile"
+description: |-
+  Get information about iam_trusted_profile
+subcategory: "IAM Identity Services"
+---
+
+# ibm_iam_trusted_profile
+
+Create, update and delete an IAM trusted profile resource. For more information, about trusted profile, see [Create a trusted profile](https://cloud.ibm.com/apidocs/iam-identity-token-api#create-profile)
+
+## Example usage
+
+```terraform
+data "ibm_iam_trusted_profile" "iam_trusted_profile" {
+	profile_id = "profile_id"
+}
+```
+
+## Argument reference
+
+Review the argument reference that you can specify for your data source.
+
+* `profile_id` - (Required, Forces new resource, String) ID of the trusted profile to get.
+
+## Attribute reference
+
+In addition to all argument references listed, you can access the following attribute references after your data source is created.
+
+* `id` - The unique identifier of the `iam_trusted_profile`.
+
+* `account_id` - (Required, String) ID of the account that this trusted profile belong to.
```

Comment:
in attribute section you shouldn't have (Required, String) ...this for all attributes